docs/graph_explorer.html

```diff
@@ -71,13 +69,16 @@
             <option>vibrant_ink</option>
             <option>xcode</option>
           </select>
-          <input id="showHelp"   type="button" value="?"/>
+          <button id="showHelp">?</button>
         </span>
-        <input id="vis_btn"  type="button" value="Vis" class="active"/>
-        <input id="tree_btn" type="button" value="Tree"/>
-        <input id="svg_btn"  type="button" value="SVG"/>
-        <input id="dot_btn"  type="button" value="DOT" class="gapAfter"/>
-        <input id="png_btn"  type="button" value="Export PNG"/>
+        <button id="vis_btn" class="active">Vis</button>
+        <button id="tree_btn">Tree</button>
+        <button id="svg_btn">SVG</button>
+        <button id="dot_btn" class="gapAfter">DOT</button>
+        <button id="png_btn" class="gapAfter">Export PNG</button>
+        <label class="small_label">Zoom
+            <input id="zoom" type="number" min="0" step="10" value="100" />
+        </label>
```

Comment:
the topbar is already really crowded, and there's a lot of other stuff going in.  i'm reluctant to add the label; i feel like this is already pretty usage clear